tests/test_prompt_template.py

```diff
@@ -126,6 +126,9 @@ def test_json2():
     template = PromptTemplate()
     with pytest.raises(MemorValidationError, match=r"Invalid template structure. It should be a JSON object with proper fields."):
```

Comment:
This test case is not incorrect, but it is not strong for this purpose; it's better to add a test case that tries to load a partially correct JSON object.